src/main/java/org/killbill/billing/plugin/qualpay/QualpayConfigProperties.java

```diff
@@ -82,4 +92,16 @@ public String getKbUsername() {
     public String getKbPassword() {
         return kbPassword;
     }
+
+    private String getClient(final String envKey, final String defaultValue) {
+        final Map<String, String> env = System.getenv();
```

Comment:
Is there a good reason to use environment variable versus per-tenant properties - same [remark](https://github.com/killbill/killbill-gocardless-example-plugin/pull/10#discussion_r1064886894) I made to Vijay's PR. Perhaps this has been already discussed and agreed upon, just trying to get some clarity.

Reply:
Environment variables are required for tests to pass while running CI (since an explicit properties file is not present. See [this](https://github.com/killbill/killbill-qualpay-plugin/blob/15a767009d3980c0201070404de6fba42dabf490/src/test/java/org/killbill/billing/plugin/qualpay/TestBase.java#L136)). I suppose we could update the tests to first upload the credentials as per-tenant properties, but this seemed simpler.